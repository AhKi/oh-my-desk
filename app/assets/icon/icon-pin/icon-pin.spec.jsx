import React from 'react';
import { shallow } from 'enzyme';

import PinIcon from './';

describe('<PinIcon />', () => {
  it('should match to snapshot when render default', () => {
    const wrapper = shallow(<PinIcon />);

    expect(wrapper).toMatchSnapshot();
  });
});
