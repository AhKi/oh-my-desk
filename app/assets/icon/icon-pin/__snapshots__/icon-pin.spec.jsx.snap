// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PinIcon /> should match to snapshot when render default 1`] = `
<svg
  height="18px"
  version="1.1"
  viewBox="0 0 17 18"
  width="17px"
  xmlns="http://www.w3.org/2000/svg"
  xmlnsXlink="http://www.w3.org/1999/xlink"
>
  <g
    fill="none"
    fillRule="evenodd"
    stroke="none"
    strokeWidth="1"
    transform="translate(-467.000000, -260.000000)"
  >
    <path
      d="M476.182646,270.194212 L476.182646,275.45698 L475.451226,276.893752 L474.719805,275.45698 L474.719805,270.194212 L469.356056,270.194212 C469.356056,270.194212 469.356056,268.6589 473.094427,268.240179 L473.094427,262.378081 C473.094427,262.378081 470.981434,262.238507 470.493821,260.842769 L480.408631,260.842769 C480.408631,260.842769 479.921017,262.238507 477.808025,262.378081 L477.808025,268.240179 C477.808025,268.240179 481.383858,268.6589 481.546396,270.194212 L476.182646,270.194212 Z"
      fill="#3C4954"
    />
  </g>
</svg>
`;
